conf/local.conf.sample

```diff
@@ -296,14 +293,16 @@ BB_DISKMON_DIRS = "\
 # Use this to add extra packages to the split-debug filesystem
 #EXTRA_DEBUG_PACKAGES = "ltrace strace"
 
-# Uncomment following two lines to enable INITRAMFS. This adds dmcrypt-initramfs-image in 
-# INITRAMFS_IMAGE if encrypted-fs image feature is enabled otherwise add core-image-minimal-initramfs.
-#INITRAMFS_IMAGE = "${@bb.utils.contains('EXTRA_IMAGE_FEATURES', 'encrypted-fs', 'dmcrypt-initramfs-image', 'core-image-minimal-initramfs', d)}"
+# Uncomment to enable encrypted-fs Machine Feature:
+#MACHINE_FEATURES_append = " encrypted-fs"
+# Uncomment it to enable INITRAMFS:
+#INITRAMFS_IMAGE = "${@bb.utils.contains('MACHINE_FEATURES', 'encrypted-fs', 'dmcrypt-initramfs-image', 'core-image-minimal-initramfs', d)}"
+# Uncomment it to bundle initramfs image with kernel image:
 #INITRAMFS_IMAGE_BUNDLE = "1"
 # removing 96boards-tools package so that rootfs does not occupy entire available space over the boot media.
-MACHINE_EXTRA_RRECOMMENDS_remove = "${@bb.utils.contains('EXTRA_IMAGE_FEATURES', 'encrypted-fs', '96boards-tools', '', d)}"
+MACHINE_EXTRA_RRECOMMENDS_remove = "${@bb.utils.contains('MACHINE_FEATURES', 'encrypted-fs', '96boards-tools', '', d)}"
```

Comment:
Why aren't we doing this through mel.conf? This is not something we're allowing the user to control I guess.

Reply:
@abelal, it was due to IMAGE_FEATURE thing. As we cannot make decisions based on IMAGE_FEATURES in a non-image recipe. Because user may add/remove an image feature.
But now, as we have moved on to MACHINE_FEATURE implementation of encrypted-fs, it can be negotiated to move these statements to somewhere else.

Reply:
mel.conf is not a recipe :)

Reply:
* non-image recipe and non-local.conf conf file. Sorry I missed. :)